artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/connect/mirror/AMQPMirrorControllerSource.java

```diff
@@ -208,6 +232,13 @@ public void sendMessage(Message message, RoutingContext context, List<MessageRef
          return;
       }
 
+      if (ignoreAddress(context.getAddress(message))) {
+         if (logger.isTraceEnabled()) {
+            logger.trace("server " + server + " is discarding send to excluded address");
```

Comment:
In addition to the "excluded" comment, for this one I'd also note it should say _which address_ is in play otherwise its not very helpful.

Reply:
Agree. I have changed both messages.